script.js

```diff
@@ -0,0 +1,16 @@
+document.querySelectorAll('li').forEach((item) => {
+  const span = document.createElement('span');
+  item.insertBefore(span, item.firstChild);
+  span.appendChild(span.nextSibling);
+});
+
+document.querySelector('.tree').addEventListener('click', (event) => {
+  if (event.target.tagName != 'SPAN') {
+    return;
+  }
```

Comment:
add empty lines after code blocks, constants blocks,
before return. It will make your code more readable